styles/globals.css

```diff
@@ -770,3 +770,45 @@ ul.check li {
 .hero-text-shadow {
   text-shadow: 0 5px 30px rgba(224,198,210,0.2);
 }
+
```

Comment:
**non-blocking**: Ideally, we'd move all of these our of global.css and use tailwind as we use elsewhere in the site. Currently `global.css` is mostly legacy CSS that we're working to shift away from over time. We don't need to do this for this PR, but we should move it at some point for consistency.

Reply:
Thank you, this make sense. I've created a [Linear](https://linear.app/inngest/issue/DEV-81/move-code-out-of-globalscss) ticket to get back to this.